go.mod

```diff
@@ -202,6 +202,7 @@ replace (
 
 	// TODO: remove when gravity update dependencies
 	github.com/peggyjv/gravity-bridge/module/v2 => github.com/crypto-org-chain/gravity-bridge/module/v2 v2.0.1-0.20220815102151-48275db7e1ee
+	github.com/tendermint/tendermint => github.com/tendermint/tendermint v0.34.24-0.20221110131553-ec471ba27efd
```

Comment:
0.34.x branch, include the check_tx cache fix.

Reply:
thanks jay [!](https://github.com/tendermint/tendermint/pull/9650/files)

Reply:
thanks @yihuang